styles/style.css

```diff
@@ -0,0 +1,382 @@
+* {
+    font-size: 1.15rem;
```

Comment:
another good thing to include in here is `box-sizing: border-box;` I'll go over it in class.